src/kvirc/ui/KviMenuBar.cpp

```diff
@@ -174,6 +176,14 @@ void KviMenuBar::setupSettingsPopup(QMenu *pop)
 	QAction *pAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Toolbar)),__tr2qs("Toolbars"));
 	pAction->setMenu(m_pToolbarsPopup);
 
+#ifndef COMPILE_ON_MAC
+	m_pMenuBarAction = opt->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::StatusBar)),__tr2qs("Show Menu Bar"),m_pFrm,SLOT(toggleMenuBar()));
```

Comment:
I dont understand the logic here, you have a button on menu bar for hiding it, once hidden theres no button to show it you relly simply on shortcuts which one has to remember...

icon wise its not a problem I can make a specific icon.